sqlalchemy_filters/loads.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy.orm import Load
+
+from .exceptions import BadLoadFormat
+from .models import Field, get_model_from_spec
+
+
+class LoadOnly(object):
+
+    def __init__(self, load_spec, query):
+        try:
+            field_names = load_spec['fields']
+        except KeyError:
+            raise BadLoadFormat(
+                '`fields` is a mandatory attribute.'
+            )
+        except TypeError:
+            raise BadLoadFormat(
+                'Load spec `{}` should be a dictionary.'.format(load_spec)
+            )
+
+        self.model = get_model_from_spec(load_spec, query)
+        self.fields = [
+            Field(self.model, field_name) for field_name in field_names
+        ]
+
+    def format_for_sqlalchemy(self):
+        return Load(self.model).load_only(
+            *[field.get_sqlalchemy_field() for field in self.fields]
+        )
+
+
+def apply_loads(query, load_spec):
+    """Apply load restrictions to a :class:`sqlalchemy.orm.Query` instance.
+
+    :param load_spec:
+        A list of dictionaries, where each item contains the fields to load
+        for each model.
+
+        Example::
+
+            load_spec = [
+                {'model': 'Foo', fields': ['id', 'name']},
+                {'model': 'Bar', 'fields': ['name']},
+            ]
```

Comment:
should we mention here that model can be left out if the query only has one model?

Reply:
Is it worth supporting the load_spec not being a list? (like we do for filters)
E.g.
`load_spec = {'model': 'Foo', 'fields': ['id', 'name']}`

It makes me wonder (if the model and list were optional), whether the `fields` entry would also be unnecessary in some cases... so in the simple one-model case you could do: 
`load_spec = ['id', 'name']`  - But maybe that's going too far?